apps/ledger-live-mobile/src/families/near/Staking/Row.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { View, StyleSheet, TouchableOpacity } from "react-native";
+import { useTranslation } from "react-i18next";
+import { NearMappedStakingPosition } from "@ledgerhq/live-common/families/near/types";
+import { Currency } from "@ledgerhq/types-cryptoassets";
+import { useTheme } from "@react-navigation/native";
+import { Text } from "@ledgerhq/native-ui";
+import { FIGMENT_NEAR_VALIDATOR_ADDRESS } from "@ledgerhq/live-common/families/near/logic";
+
+import CounterValue from "../../../components/CounterValue";
+import ArrowRight from "../../../icons/ArrowRight";
+import LText from "../../../components/LText";
+import ValidatorImage from "../shared/ValidatorImage";
+
+type Props = {
+  stakingPosition: NearMappedStakingPosition;
+  currency: Currency;
+  onPress: (_: NearMappedStakingPosition) => void;
+  isLast?: boolean;
+};
+
+export default function StakingRow({
+  stakingPosition,
+  currency,
+  onPress,
+  isLast = false,
+}: Props) {
+  const { colors } = useTheme();
+  const { t } = useTranslation();
+  const { validatorId, formattedAmount, staked } = stakingPosition;
+
+  return (
+    <TouchableOpacity
+      style={[
+        styles.row,
+        styles.wrapper,
+        !isLast
+          ? { ...styles.borderBottom, borderBottomColor: colors.lightGrey }
+          : undefined,
+      ]}
+      onPress={() => onPress(stakingPosition)}
+    >
+      <View style={[styles.icon]}>
+        <ValidatorImage
+          size={42}
+          isLedger={validatorId === FIGMENT_NEAR_VALIDATOR_ADDRESS}
+          name={validatorId ?? ""}
+        />
+      </View>
+
+      <View style={styles.nameWrapper}>
+        <Text variant={"body"} fontWeight={"semiBold"} numberOfLines={1}>
+          {validatorId}
+        </Text>
+
+        <View style={styles.row}>
+          <LText style={styles.seeMore} color="live">
```

Comment:
Nope, decision was inherited from `apps/ledger-live-mobile/src/families/cosmos/Delegations/Row.tsx` - changed so that we are only using `Text` in this file.